### PR TITLE
update sum front

### PR DESCRIPTION
**Pull request by lilyanB**

### CODE REVIEW

**Comment by gregLibert** [comment]
@lilyanB can you fix the 4 code smells?

**Comment by lilyanB** [comment]
> you could already mock the protobuf caller so that we just have to add it. this is what it should look like:
> 
> The import: `import { sum } from "./sumCaller"`
> 
> how you call it: `sum(bigint1, bigint2, coins)` (coins is the amount you are willing to pay to the block creator, also a bigint)
> 
> the functions returns an any type (it should correspond to the outputs type of the smart contract function)

done

File: sum/front/src/utils/sumCaller.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export async function sum(
```
Comment (Elli610) [inline sum/front/src/utils/sumCaller.ts:1]: maybe you could add a little comment explaining why this function is mocked for now

**Review by Elli610 — APPROVED** [review]
in sum/front/src/const/const.js maybe you shold add a quick comment about

File: sum/front/src/utils/sumCaller.ts (hunk 1 above)
Comment (lilyanB) [reply sum/front/src/utils/sumCaller.ts:4]: sure
done

**Review by Elli610 — COMMENTED** [review]
can you solve the bugs ?

**Review by Elli610 — CHANGES_REQUESTED** [review]
when I run npm i and npm run start, I get some warnings related to the  wallet provider. here a some of them:

`WARNING in ./node_modules/@massalabs/wallet-provider/dist/utils/time.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from 'Massa\massa-sc-examples\sum\front\node_modules\@massalabs\wallet-provider\src\utils\time.ts' file: Error: ENOENT: no such file or directory, open 'Massa\massa-sc-examples\sum\front\node_modules\@massalabs\wallet-provider\src\utils\time.ts'`

`WARNING in ./node_modules/@massalabs/wallet-provider/dist/account/index.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from 'Massa\massa-sc-examples\sum\front\node_modules\@massalabs\wallet-provider\src\account\index.ts' file: Error: ENOENT: no such file or directory, open 'Massa\massa-sc-examples\sum\front\node_modules\@massalabs\wallet-provider\src\account\index.ts'
`

There a re like a dozen of them. Can you investigate ?

**Review by Elli610 — APPROVED** [review]

**Review by Ben-Rey — APPROVED** [review]

**Pull request merged**